src/app/profile/settings/feature-opt-in/feature-opt-in.component.html

```diff
@@ -1,86 +1,126 @@
-                <div>
-                  <p class="feature-title">Beta Features</p>
-                  <p class="feature-description">Experience various features that are ready for beta testing.</p>
-                </div>
+            </div>
+            <div class="col-xs-9 col-sm-5">
+              <div class="feature-details-description">
+                <ul *ngFor="let feature of item.features">
+                  <li><strong>{{feature?.attributes?.name}}</strong>: {{feature?.attributes?.description}}.</li>
+                </ul>
               </div>
             </div>
           </div>
-        </div>
-      </label>
-      <label class="feature-opt-in-label" for="experimental">
-        <div class="row">
-          <div class="list-group-item">
-            <div class="list-view-pf-checkbox">
-              <input id="experimental" type="radio" name="Experimental Features" value="experimental" [(ngModel)]="featureLevel" (ngModelChange)="updateProfile()">
+
+          <div *ngIf="item.expandId === 'experimental'">
+            <div class="col-sm-1">
+              <span class="fa {{item.icon}} list-pf-icon icon-big"></span>
             </div>
-            <div class="list-view-pf-main-info">
-              <div class="list-view-pf-left">
-                <span class="fa fa-flask list-view-pf-icon-sm"></span>
+            <div class="col-sm-4">
+              <div class="feature-details-title">
+                There are currently {{item.features.length}} {{item.name}} features
               </div>
-              <div class="list-view-pf-body">
-                <div>
-                  <p class="feature-title">Experimental Features</p>
-                  <p class="feature-description">This is the default and current release version of the product.</p>
-                </div>
+              <div class="feature-details-description">
+                These features are currently in experimental testing and have no guarantee of performance or stability.<br/>
+                Use these at your own risk.
+              </div>
+            </div>
+            <div class="col-xs-9 col-sm-5">
+              <div class="feature-details-description">
+                <ul *ngFor="let feature of item.features">
+                  <li><strong>{{feature?.attributes?.name}}</strong>: {{feature?.attributes?.description}}.</li>
+                </ul>
               </div>
             </div>
           </div>
-        </div>
-      </label>
-      <label class="feature-opt-in-label" for="internal">
-        <div class="row">
-          <div class="list-group-item">
-            <div class="list-view-pf-checkbox">
-              <input id="internal" type="radio" name="Internal Features" value="internal" [(ngModel)]="featureLevel" (ngModelChange)="updateProfile()">
+
+          <div *ngIf="item.expandId === 'internal'">
+            <div class="col-sm-1">
+              <span class="fa {{item.icon}} list-pf-icon icon-big"></span>
```

Comment:
instead of using `icon-big` to define the size of the icon, can you use the built-in Font Awesome sizing?

From https://fontawesome.com/v4.7.0/examples/:
`To increase icon sizes relative to their container, use the fa-lg (33% increase), fa-2x, fa-3x, fa-4x, or fa-5x classes.`
`https://github.com/FortAwesome/Font-Awesome/blob/v4.7.0/less/larger.less`